docs/source/releases/latest/south-china-sea-sector.yaml

```diff
@@ -0,0 +1,34 @@
+enhancement:
```

Comment:
Why is this release note / branch named "south china sea sector" ?  Should the release note have a more informative name?

Reply:
I renamed this.